kolibri/plugins/media_player/assets/src/views/MediaPlayerIndex.vue

```diff
@@ -188,6 +188,14 @@
       transcriptVisible() {
         return this.transcript && !this.loading && this.captionTracks.length > 0;
       },
+      /* eslint-disable kolibri/vue-no-unused-properties */
+      /**
+       * @public
+       */
+      defaultDuration() {
+        return this.duration || this.player.duration();
```

Comment:
Could `this.player` not be initialized by the time this is executed? The player is instantiated on `mount` and then `$nextTick`